homeassistant/components/media_player/ziggo_mediabox_xl.py

```diff
@@ -0,0 +1,218 @@
+"""
+Support for interface with a Ziggo Mediabox XL.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/media_player.ziggo_mediabox_xl/
+"""
+import logging
+import socket
+import requests
+
+import voluptuous as vol
+
+from homeassistant.components.media_player import (
+    PLATFORM_SCHEMA, MediaPlayerDevice,
+    SUPPORT_TURN_ON, SUPPORT_TURN_OFF,
+    SUPPORT_NEXT_TRACK, SUPPORT_PREVIOUS_TRACK, SUPPORT_SELECT_SOURCE,
+    SUPPORT_PLAY, SUPPORT_PAUSE)
+from homeassistant.const import (
+    CONF_HOST, CONF_NAME, STATE_OFF, STATE_ON, STATE_UNKNOWN,
+    STATE_PAUSED, STATE_PLAYING)
+import homeassistant.helpers.config_validation as cv
+
+_LOGGER = logging.getLogger(__name__)
+
+KNOWN_DEVICES_KEY = 'ziggo_mediabox_xl_known_devices'
```

Comment:
**Preference:**
Our normal standard is to prefix `hass.data` keys with `DATA_`. Can we rename `KNOWN_DEVICES_KEY` to `DATA_KNOWN_DEVICES`?

Reply:
Done